src/condor_shadow.V6.1/remoteresource.cpp

```diff
@@ -2268,6 +2268,8 @@ RemoteResource::initFileTransfer()
 	const char * currentFile = NULL;
 	Directory spoolDirectory( spoolPath.c_str() );
 	while( (currentFile = spoolDirectory.Next()) ) {
+		// getNumberFromFileName() ignores FAILURE files, which is
+		// exactly the behavior we want here.
 		int manifestNumber = manifest::getNumberFromFileName( currentFile );
 		if( manifestNumber > largestManifestNumber ) {
 			largestManifestNumber = manifestNumber;
```

Comment:
We couldn't use `std::max_element` here?

Reply:
The non-MANIFEST files in that directory screw up the comparison operator.